Stationeers.VS.props

```diff
@@ -0,0 +1,50 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)Stationeers.VS.User.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net4.8</TargetFramework>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <LangVersion>9.0</LangVersion>
+    <ErrorReport>none</ErrorReport>
+    <LanguageTargets>$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Targets>AnyCPU</Targets>
+    <OutputPath>$(SolutionDir)bin\$(Configuration)</OutputPath>
+
+    <!-- Do Not Change Anything Below -->
+    <StationeersDirectory>$(SteamLibraryDirectory)\Stationeers</StationeersDirectory>
+    <StationeersDataDirectory>$(StationeersDirectory)\rocketstation_Data</StationeersDataDirectory>
+    <StationeersManagedDirectory>$(StationeersDataDirectory)\Managed</StationeersManagedDirectory>
+    <StationeersPluginsDirectory>$(StationeersDataDirectory)\Plugins</StationeersPluginsDirectory>
+    <StationeersResourcesDirectory>$(StationeersDataDirectory)\Resources</StationeersResourcesDirectory>
+    <StationeersStreamingAssetsDirectory>$(StationeersDataDirectory)\StreamingAssets</StationeersStreamingAssetsDirectory>
+    <StationeersDocumentsDirectory>$(HOMEDRIVE)$(HOMEPATH)\Documents\My Games\Stationeers</StationeersDocumentsDirectory>
```

Comment:
Same here.  I use relative paths to keep the mod data isolated to the game folder.
ex: `<StationeersDirectory>E:\Stationeers\beta\</StationeersDirectory>
    <StationeersDocumentsDirectory>E:\Stationeers\beta\config</StationeersDocumentsDirectory>`